src/Plugin/Block/SubsitesHierarchyTrait.php

```diff
@@ -114,6 +114,7 @@ protected function getRootId(EntityInterface $entity): ?int {
         'localgov_subsites_overview',
         'localgov_subsites_page',
       ], TRUE)
+      && is_numeric($entity->id())
```

Comment:
Would changing to is_null work if it could be a string?

```suggestion
      && !is_null($entity->id())
```

Reply:
We can try that, if that would be more correct.